.github/workflows/run-unittest.yml

```diff
@@ -0,0 +1,26 @@
+name: CBI unittest
+
+on: [push]
```

Comment:
Should we also limit the paths here?  I think we probably only need to run the tests if something in `bin/*`, `codebasin/**`, `setup.py` or `MANIFEST.in` changes.  Without limiting the paths, won't we also run all the tests when we change something like the documentation?